components/BlurryPoster/BlurryPoster.tsx

```diff
@@ -43,6 +43,10 @@ type BlurryPosterProps = HTMLAttributes<HTMLDivElement> & {
   alt?: string;
   tabColor?: AccentColorType;
   cta?: React.ReactNode;
+  /**
+   * This isn't displayed in the component, aria-describedby will be added to the images if a caption exists
+   */
+  caption?: string;
```

Comment:
Why not label this `ariadescribedby` then?

Reply:
The caption isn't actually the text that is displayed in the BlurryPoster (doubles up as the top part of the story hero sans caption/textured bar) nor for the `aria-describedby` attribute. I'm only passing the caption prop in (if there is a caption passed from the StoryMVP page) so I can do a boolean check on whether a caption exists. If there is one, I add `aria-describedby="story-hero-caption"` (the id of the caption which is always unique) to all the images in the hero.

Reply:
Like this - just to tell the screen reader that the hero image has a caption associated with it.

![Screenshot 2023-11-01 at 1 56 09 PM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/0dd9ab8a-368a-48b2-bd52-0333ef4a4183)

Reply:
ah thanks for splainin.